pandas/core/internals.py

```diff
@@ -2921,6 +2914,7 @@ def sparse_reindex(self, new_index):
                                           placement=self.mgr_locs)
 
 
+# TODO: deprecate `fastpath` kwarg after getting pyarrow to stop passing it
```

Comment:
instead of the comment. post a JIRA to arrow (fix even better). pass this as ``fastpath=None`` and you can show a DeprecationWarning if ``fastpath`` is not ``None``.